azure-iot-device/azure/iot/device/iothub/sync_clients.py

```diff
@@ -366,26 +368,56 @@ def receive_message(self, block=True, timeout=None):
         logger.info("Message received")
         return message
 
+    def get_storage_info(self, blob_name):
```

Comment:
Is this a clear enough name on the API? `get_storage_info_for_blob` or something to show the connection between this and the other blob method below?